modules/tinymce/src/themes/silver/test/ts/browser/view/ViewTest.ts

```diff
@@ -318,6 +318,29 @@ describe('browser.tinymce.themes.silver.view.ViewTest', () => {
       toggleView('myview1');
       TinyAssertions.assertCursor(editor, [ 0, 0 ], 1);
     });
+
+    it('TINY-9671: should be possible to navigate the header via keyboard', async () => {
+      const editor = hook.editor();
+      const root = SugarShadowDom.getRootNode(TinyDom.targetElement(editor));
+      toggleView('myview1');
+      FocusTools.setFocus(root, '.tox-view__header');
+      await FocusTools.pTryOnSelector('Focus should be on the view header', root, '.tox-view__header');
+
+      TinyUiActions.keystroke(editor, Keys.enter());
```

Comment:
Is this keystroke needed. Maybe you could just move focus to the first button using setFocus then check if arrows work?

Reply:
I did it because users need to do this to navigate the header, I can remove it but I think it could be useful if someone for example remove the possibility to focus the header and it became not navigable